.github/pull_request_template.md

```diff
@@ -10,6 +10,7 @@
 - [ ] New functionality has been documented.
   - [ ] New functionality has javadoc added
 - [ ] Commits are signed per the DCO using --signoff 
+- [ ] Commit changes are listed out in CHANGELOG.md file (See: [Changelog](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#changelog))
```

Comment:
This link can be relative to this project?

Reply:
Updated it to be relative.